proto/leaderboard.proto

```diff
@@ -130,64 +130,64 @@ enum _Order {
 }
 
 message _DeleteLeaderboardRequest {
-  string cache_name = 1;
   string leaderboard = 2;
+  reserved 1;
```

Comment:
idk if we have a best practice around this, but I wonder if, for historical purposes, it would be useful to leave comments when we make these reservations, so we know what the fields were originally allocated for?